Procfile

```
@@ -1,5 +1,5 @@
 release: bin/release
 web: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-prod.conf.py warehouse.wsgi:application
 web-uploads: bin/start-web ddtrace-run python -m gunicorn.app.wsgiapp -c gunicorn-uploads.conf.py warehouse.wsgi:application
-worker: bin/start-worker celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
-worker-beat: bin/start-worker celery -A warehouse beat -S redbeat.RedBeatScheduler -l info
+worker: bin/start-worker ddtrace-run celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
```

Cabotage does allow setting envs via the Procfile, as defined in this spec: https://smartmob-rfc.readthedocs.io/en/latest/1-procfile.html#formal-specification

So you could update:

```suggestion
worker: DD_SERVICE=warehouse-worker bin/start-worker ddtrace-run celery -A warehouse worker -Q default -l info --max-tasks-per-child 32
```

---

Thanks, TIL! Updated.